### PR TITLE
Project restructure

### PR DESCRIPTION
**Pull request by GreenStage**
Proposal to solve https://github.com/bancodobrasil/stop-analyzing-api/issues/11

Lets discuss this :D

Thanks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-prisma/migrations*
-src/db/db_gen*
-stop-analyzing-api
\ No newline at end of file
+scripts/prisma/migrations*
+internal/db/db_gen*
+stop-analyzing-api
+.vscode/
```
Comment (tiagostutz) [inline .gitignore:4]: What about putting the .idea in the .gitignore as well?

File: cmd/migrate.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	"github.com/bancodobrasil/stop-analyzing-api/migration"
+	"github.com/bancodobrasil/stop-analyzing-api/internal/migration"
```
Comment (tiagostutz) [inline cmd/migrate.go:4]: Nice!

File: internal/api/service.go [hunk 3]
```diff
@@ -1,30 +1,29 @@
-package service
+package api
 
 import (
+	v12 "github.com/bancodobrasil/stop-analyzing-api/internal/api/v1"
```
Comment (tiagostutz) [inline internal/api/service.go:4]: Is the number "2" a typo? Maybe it was supposed to be "v1"?

Comment (GreenStage) [reply internal/api/service.go]: fixed in https://github.com/bancodobrasil/stop-analyzing-api/pull/21/commits/544ac955d77f285785fadbd0b5985d31c76583d5

File: .gitignore (hunk 1 above)
Comment (GreenStage) [reply .gitignore]: fixed in https://github.com/bancodobrasil/stop-analyzing-api/pull/21/commits/aa499c82ec50ab842ad82a05f85f1fc2fd28c9ac

**Comment by GreenStage** [comment]
From what I could gather from the [standard project layout](https://github.com/golang-standards/project-layout), usually people use the cmd folder for multi-app projects. 

Here's an example: https://github.com/influxdata/influxdb/tree/master/cmd

In our case, this implies having _cmd/stop-analyzing-api_ and _cmd/migrate_. Both these folders have a main, and therefore are executable independently.

Is this something we want to pursue, or just leave it as it is?

**Comment by tiagostutz** [comment]
I think we can leave it as is and as the migrator grows and gets more complex we can address this restructure in an specific issue.

This PR is  ready to go! Thanks for this great job!

**Pull request merged**